guardian/typerighter
# Add scalafmtCheckAll step to CI

## What does this change?

This adds a `scalafmtCheckAll` to our GitHub Actions-based CI.

It also formats the project to make sure all existing files are formatted in line with `scalaFmt`. (The **only** non-formatting change is to `.github/workflows/ci.yaml`)

## How to test

1. Run `sbt scalafmtCheckAll` locally (with the app running in another terminal via `./script/start`). Does the formatting check pass?
2. Look at a CI build for this PR in the Github Actions tab. Does the formatting check pass? Does the CI otherwise perform as expected?
